customext/cflush.cc

```diff
@@ -10,17 +10,17 @@ struct : public arg_t {
 
 static reg_t custom_cflush(processor_t* p, insn_t insn, reg_t pc)
 {
```

Comment:
Commit message for 289e47fdb51ae6d97b202fdf66cceb9fbe305ee1 is misleading because it updates more than just the rocc extension. Also: update for what? How about: "Update existing extensions for new API"